optimum/exporters/openvino/__main__.py

```diff
@@ -105,6 +113,75 @@ def infer_task(
                 )
     return task
 
+def eagle3_config(model_path: str):
+    config_file = os.path.join(model_path, 'config.json')
+    # rename the origin config
+    new_config_file = os.path.join(model_path, 'config_temp.json')
+    shutil.copy2(config_file, new_config_file)
+
+    # read config
+    with open(new_config_file, 'r', encoding='utf-8') as f:
+        config = json.load(f)
+    # modify config
+    if 'model_type' in config.keys():
+        org_type = config['model_type']
+        if org_type != 'llama':
+            raise ValueError(f"Currently eagle3 does not support model_type={org_type}, it only supports the conversion of llama-based draft models.")
+    # CVS-174959
+    if 'tie_word_embeddings' in config.keys():
+        if config['tie_word_embeddings']:
+            config['tie_word_embeddings'] = False
+    moduler_name = 'optimum.exporters.openvino.model_patcher'
+    spec = importlib.util.find_spec(moduler_name)
+    if spec and spec.origin:
+        moduler_path = os.path.dirname(spec.origin)
+        config['auto_map'] = {
+            "AutoModel": moduler_path + "--model_patcher.LlamaEagle3Model",
+            "AutoModelForCausalLM": moduler_path + "--model_patcher.LlamaEagle3ForCausalLM"
+        }
+    # write new config.json
+    with open(new_config_file, 'w', encoding='utf-8') as f:
+        json.dump(config, f, ensure_ascii=False, indent=2)
+    return os.path.abspath(new_config_file)
+
+def extract_d2t(model_path: str, output_path: str):
+    load_model_path=os.path.join(model_path, "pytorch_model.bin")
+    output_path = os.path.join(output_path, "eagle3.safetensors")
```

Comment:
optimum-cli must not generate safetensors. This is not optimum-cli responsibility.